assembly/program.test.ts

```
@@ -0,0 +1,155 @@
+    assert.isEqual(
+      jumpTable.toString(),
+      "JumpTable[0 -> 18446744073709551615, 1 -> 18446744073709551615, 2 -> 18446744073709551615, ]",
+    );
+    return assert;
+  }),
+];
+
+function opcode(search: InstructionRun): number {
+  const idx = RUN.indexOf(search);
+  if (idx < 0) {
+    throw new Error(`Opcode not found in RUN table for instruction`);
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)